port/user_settings.h

```diff
@@ -44,6 +44,9 @@
 
 #define HAVE_SNI
 
+#define HAVE_OCSP
+#define HAVE_CERTIFICATE_STATUS_REQUEST
```

Comment:
I think the first option enables OCSPv1 support in wolfSSL and the second one enables the certificate checking within wolfSSLs certificate manager. We definitely do not want to have these enabled by default. (Because most servers still don't support OCSP.) And hence I defaulted the CONFIG_WOLFSSL_HAVE_OCSP by default in the Kconfig.